cg/apps/slurm/sbatch.py

```diff
@@ -9,6 +9,7 @@
 #SBATCH --mail-user={email}
 #SBATCH --time={hours}:{minutes}:00
 #SBATCH --qos={priority}
+#SBATCH --exclude=gpu-compute-0-[0-1]
```

Comment:
I'd suggest to move this into constants. To be able to override or change it easier later on